index.html

```diff
@@ -13,226 +13,328 @@
 
 <body>
 
-  <div class="header">
+  <div class="container">
 
-    <div class="time">
-      <img class="icons image1" src="images/Time.png" alt="time">
-    </div>
+    <header class="header">
 
-    <div class="connection">
-      <img class="icons image2" src="images/Cellular-Connection.png" alt="Cellular_Connection-path">
-      <img class="icons image3" src="images/Wifi.png" alt="wifi">
-      <img class="icons image4" src="images//Capacity.png" alt="Capacity">
-    </div>
+      <div class="logo">
+        <a href="#">MUKUNZI</a>
+      </div>
 
-  </div>
+      <nav>
+        <img class="vector" src="images/Vector.png" alt="vector">
+        <img class="vector" src="images/Vector.png" alt="vector">
+        <img class="vector" src="images/Vector.png" alt="vector">
+      </nav>
 
-  <div class="name1">
+    </header>
 
-    <div class="nam1-text">
-      <h2 class="joe">Joe Doe</h2>
-    </div>
 
-    <div class="name1-img">
-      <img class="vector" src="images/Vector.png" alt="vector">
-      <img class="vector" src="images/Vector.png" alt="vector">
-      <img class="vector" src="images/Vector.png" alt="vector">
-    </div>
-  </div>
+    <div class="main">
 
+      <div class="presentation">
 
-  <div class="main">
+        <div class="presentation1">
 
-    <div class="name2">
-      <div class="laura laura1">
-        <img src="images/Ellipse18.svg" alt="elipse">
-      </div>
+          <div class="my-name">
+            <h1>Patrick MUKUNZI</h1>
+          </div>
 
-      <div class="laura laura2">
-        <h1 class="laura-name"> Laura Jong</h1>
+          <div class="my-job">
+            <h2>Software Developer</h2>
+          </div>
 
-        <div class="job">
-          <h2>Software Developper</h2>
         </div>
-      </div>
-    </div>
 
+        <div class="presantation2">
 
-    <div class="description">
-      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
-        what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    </div>
+          <div class="job-description">
+
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+          </div>
+
+          <div class="knowledge">
+
```

Comment:
- [ ]  Please stick with the color of the bar presented in the design 

![image](https://user-images.githubusercontent.com/74744787/158881059-f22c0a84-d72f-40fa-925d-0aa4b11a604e.png)

![image](https://user-images.githubusercontent.com/74744787/158881123-6347b070-42a6-42cd-ba30-d13091029cc5.png)

Reply:
resolved 👍